scheduling/python/interval-scheduling.py

```diff
@@ -5,29 +5,37 @@
 Algorithm Type: Greedy
 Time Complexity: O(n*log(n))
 """
+jobs = [
+  (0, 2, 8),  # (job_id, start_time, finish_time)
+  (1, 6, 10),
+  (2, 1, 3),
+  (3, 4, 7),
+  (4, 3, 6),
+  (5, 1, 2),
+  (6, 8, 10),
+  (7, 10, 15),
+  (8, 12, 16),
+  (9, 14, 16)
+]
+
 
-def get_opt_schedule(jobs): # Returns the job_id's in the optimal schedule
+def get_opt_schedule(jobs):
+  """
+  Return the job_id's in the optimal jobs
+  https://en.wikipedia.org/wiki/Interval_scheduling#Greedy_polynomial_solution
+
+  >>> get_opt_schedule(jobs)  # Result shows four batches but which jobs in each batch?
```

Comment:
> Result shows four batches but which jobs in each batch?

What do you mean by this? The algorithm finds the max possible number of non overlapping jobs and returns their job id's.